src/wazuh_db/wdb_integrity.c

```diff
@@ -584,13 +584,16 @@ int wdbi_check_sync_status(wdb_t *wdb, wdb_component_t component) {
     if ( cJSON_IsNumber(j_last_attempt) && cJSON_IsNumber(j_last_completion) && cJSON_IsString(j_checksum)) {
         int last_attempt = j_last_attempt->valueint;
         int last_completion = j_last_completion->valueint;
+        char *checksum = cJSON_GetStringValue(j_checksum);
 
         // Return 0 if there was not at least one successful syncronization or
-        // if the syncronization is in process or there was an error iun the syncronization
-        if (result = (last_completion != 0 && last_attempt <= last_completion ), result) {
+        // if the syncronization is in process or there was an error in the syncronization
```

Comment:
Maybe we can change this comment with an explanation like:
// The component is considered as synced if there is at least one complete synchronization that updated last_completion accordingly.